gradle/dependencies.gradle

```diff
@@ -47,36 +47,39 @@ if (hasProperty('scalaVersion')) {
 versions["baseScala"] = versions.scala.substring(0, versions.scala.lastIndexOf("."))
 
 versions += [
+  activation: "1.1.1",
   apacheda: "1.0.0",
   apacheds: "2.0.0-M24",
   argparse4j: "0.7.0",
-  bcpkix: "1.58",
+  bcpkix: "1.59",
```

Comment:
Do the library versions need to be bumped as part of this fix, or is this just a convenient time to do it?